wheelhouse.txt

```diff
@@ -8,21 +8,21 @@ pip<22.1;python_version >= '3.8'
 # for trusty
 Jinja2==2.10;python_version >= '3.0' and python_version <= '3.4' # py3 trusty
 Jinja2==2.11;python_version == '2.7' or python_version == '3.5'  # py27, py35
-Jinja2;python_version >= '3.6' # py36 and on
+# Focal failures - https://github.com/juju-solutions/layer-basic/issues/210
+Jinja2==2.11;python_version >= '3.6' and python_version <= '3.8' # py38, focal
+Jinja2;python_version > '3.8'
 
 PyYAML==5.2;python_version >= '3.0' and python_version <= '3.4' # py3 trusty
 PyYAML<5.4;python_version == '2.7' or python_version >= '3.5'  # all else
 
 MarkupSafe<2.0.0;python_version < '3.6'
-MarkupSafe<2.1.0;python_version == '3.6' # Just for python 3.6
-MarkupSafe;python_version >= '3.7' # newer pythons
+MarkupSafe<2.1.0;python_version >= '3.6' and python_version <= '3.8' # py38, focal
+MarkupSafe;python_version > '3.8'
 
-setuptools<42;python_version < '3.8'
-# https://github.com/juju-solutions/layer-basic/issues/201
-setuptools<62.2.0;python_version >= '3.8'
-setuptools-scm<=1.17.0;python_version < '3.8'
-# https://github.com/pypa/setuptools_scm/issues/722
-setuptools-scm<7;python_version >= '3.8'
+setuptools<42;python_version <= '3.8'
+setuptools;python_version > '3.8'
+setuptools-scm<=1.17.0;python_version <= '3.8'
+setuptools-scm;python_version > '3.8'
```

Comment:
There's some major pin changes here for the versions.  e.g. unlocking setuptools and setuptools-scm for anything > 3.8 and changing the pin of 3.8.  Lots of charms are currently built and work correctly with the current set of pins, so there needs to be a really good reason why this should change, especially as could break existing stable charm releases if they were rebuilt?